scss/utilities/_borders.scss

```diff
@@ -54,6 +54,10 @@
   border-radius: 50% !important;
 }
 
+.rounded-pill {
+  border-radius: 1.5rem !important;
```

Comment:
Any reason why you don't use the mixin?

Reply:
@XhmikosR  I tried using the mixin that I made, I wanted to specify `!important` as it's a utility class as well and wanted to follow a similar format to the above utilities, however, I couldn't use `!important` in the mixin file as it failed the linter and wouldn't let me build:

```
Unexpected !important (declaration-no-important) 
```
So I had to use it without the mixin here.

Reply:
Using `!important` is bad, and I wouldn't normally use it in projects. However, from my understanding Bootstrap 4.x uses this to ensure they work.

Reply:
Yeah, definitely use the mixin. Just disable the check for this line only.